scripts/packages/chocolatey/tools/chocolateyinstall.ps1

```diff
@@ -3,16 +3,18 @@ $packageName = 'bazel'
 
 $toolsDir = Split-Path -parent $MyInvocation.MyCommand.Definition
 $paramsText = get-content "$($toolsDir)\params.txt"
```

Comment:
`get-content` already returns an array of strings, but it sounds like #2671 problem is that powershell v2 treats them as `system.object` rather than `system.string`. Which is super nice.